src/post/Header.tsx

```diff
@@ -306,19 +310,26 @@ const PostHeader: React.FC<PostHeaderProps> = (props) => {
           top: 50%;
           transform: translateY(-50%);
         }
+        .avatar.mask {
+          margin-right: 10%;
+        }
+        .avatar.mask::before {
+          mask-image: url(${AvatarMask});
+          mask-position: center;
+          mask-repeat: no-repeat;
+        }
         .secret-avatar {
           position: absolute;
           bottom: 0;
-          right: 0;
-          width: 40%;
-          height: 40%;
+          right: -42%;
+          width: 64%;
```

Comment:
Actually, disregard this comment. I will refactor this to use a couple of fixed sizes rather than percentages so everything flows better. I'll take care of it since I need to think about what those sizes should be.